go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/saucelabs/forwarder
 
-go 1.23
+go 1.23.1
```

Comment:
GO_VERSION=1.23.2

Reply:
Ran go mod tidy on 1.23.2 same result.

Reply:
It's the minimal go version that can build this package.